ci/create_experment.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+
+'''Basic python script to create an experment directory on the fly from a given
+
+yaml file for the arguments to the two scripts below in ${HOMEgfs}/workflow
+
+where ${HOMEgfs} is specified within the input yaml file.
+
+${HOMEgfs}/workflow/setup_expt.py
+${HOMEgfs}/workflow/setup_xml.py
+'''
+
+import os, sys, socket
+import glob
+from pathlib import Path
+
+from pygw.yaml_file import YAMLFile
+from pygw.logger import Logger
+from pygw.executable import Executable
+
+from argparse import ArgumentParser, ArgumentDefaultsHelpFormatter
+
+from hosts import Host
+
+logger = Logger(level='DEBUG', colored_log=True)
+_here = Path.absolute(Path(__file__)).parents[0]
+_top  = Path.absolute(Path(__file__)).parents[1]
+
+def input_args():
+    """
+    input_args()
+    
+    Method to collect user arguments for `create_experment.py`
```

Comment:
```suggestion
    Method to collect user arguments for `create_experiment.py`
```

Reply:
oops, still had one more, should have "grep-ed" for it